### PR TITLE
Minor tweaks to DDR menu docs

### PR DESCRIPTION
**Pull request by Timo-Breumelhof**

### CODE REVIEW

File: content/designers/theme-objects/ddrmenu/index.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ links: ["[DNN Wiki: DotNetNuke Skins](https://www.dnnsoftware.com/wiki/dotnetnuk
 
 # DDRMenu Overview  
 
-Originally built and maintained by [Mark Allan @MarkXA](https://twitter.com/markxa), the DDRMenu is a robut menu system that is installed and used by default in DNN since version 6.0.  
+Originally built and maintained by [Mark Allan @MarkXA](https://twitter.com/markxa), the DDRMenu is a robust menu system that is installed and used by default in DNN since version 6.0.
+With the right Template and Styling any menu style can be created, with DDR menu you can use [TOKEN], XSLT or Razor templates.
```
Comment (dnndev) [inline content/designers/theme-objects/ddrmenu/index.md]: `style can be created, with DDR menu` should be `style can be created. with DDR menu`

File: content/designers/theme-objects/ddrmenu/index.md [hunk 2]
```diff
@@ -128,4 +129,5 @@ Module options are set through the standard module settings.  Template arguments
 
 *   [DDRMenu Razor Templates in DotNetNuke 7.x](http://www.dnnsoftware.com/Resources/Blogs/EntryId/3619/ContentItemId/150607/DDRMenu-Razor-Templates-in-DotNetNuke-7-x.aspx)  
 *   [Advanced Menu Design with DDRMenu](http://www.dnnsoftware.com/Resources/Blogs/EntryId/3446/Advanced-Menu-Design-with-DDRMenu.aspx)  
-*   [A Razor Templated DDRMenu for DotNetNuke 6](http://www.aubrett.com/InformationTechnology/WebDevelopment/CMSPlatforms/DotNetNuke/RazorTemplatedDDRMenu.aspx)  
\ No newline at end of file
+*   [A Razor Templated DDRMenu for DotNetNuke 6](http://www.aubrett.com/InformationTechnology/WebDevelopment/CMSPlatforms/DotNetNuke/RazorTemplatedDDRMenu.aspx)  
+*   [DDR Menu Demo skins](https://demo.40fingers.net.local/dnn-ddr-demo-skin)
```
Comment (dnndev) [inline content/designers/theme-objects/ddrmenu/index.md]: The 40fingers URL does not resolve

**Comment by Timo-Breumelhof** [comment]
ok sorry clicked the wrong button. Ill fix

**Comment by Timo-Breumelhof** [comment]
@dnndev the second commit contains the changes. I'll take a bit more time to review next time...

**Pull request merged**